build_tools/github_actions/post_build_upload.py

```diff
@@ -285,12 +272,6 @@ def run(args):
         help="AMDGPU family name (default: $AMDGPU_FAMILIES)",
     )
     parser.add_argument("--run-id", type=str, help="GitHub run ID of this workflow run")
-    parser.add_argument(
-        "--ci",
-        default=False,
-        help="Enable CI steps",
-        action=argparse.BooleanOptionalAction,
-    )
     parser.add_argument(
         "--upload",
         default=False,
```

Comment:
Ah, now I see the code flow with `--ci`. Looks like it is always used like so:
```python
if is_ci and args.upload
```

How about changing the default to `--upload` instead of having a separate option?

```python
is_ci = os.getenv("CI", False)  # maybe explicitly convert to boolean here
parser.add_argument(
    "--upload",
    default=is_ci,
```

Reply:
I don't think that's sufficient, but it should work well enough as currently used in CI.

If I run the script locally with `set CI=0`, that will be `"0"` which is truthy. I would need to run with `set CI=` or `unset CI` to get a falsey value.